docs/how-to/how-to-hotplug-memory-arm64.md

```diff
@@ -15,6 +15,18 @@ $ sudo .ci/aarch64/install_rom_aarch64.sh
 $ popd
 ```
 
+## Config KATA QEMU
+
+After executing the above script, two files will be generated under the directory `/usr/share/kata-containers/` by default, namely `kata-flash0.img` and `kata-flash1.img`. Next we need to change the configuration file of `kata qemu`, which is in `/opt/kata/share/defaults/kata-containers/configuration-qemu.toml` by default, specify in the configuration file to use the UEFI ROM installed above. Please refer to the following configuration.
```

Comment:
Hi @singhwang -, I my ubuntu machine, the kata config file is "/usr/share/defaults/kata-containers/configuration-qemu.toml". This file may vary with distro but I have not checked for that. We can use "kata-runtime env" to find the right path.

Reply:
I know what you said, but for general use, I installed it using kata deploy, and the path is unified on all distributions.

Reply:
Do you mind adding a sentence like "if no, check it for kata-runtime env" after 

> which is in `/opt/kata/share/defaults/kata-containers/configuration-qemu.toml` by default

I think it's better to serve different senario.

Reply:
@jongwu I can add it, but I have to resubmit another pr on the stable-3.0 branch, can you review it for me？

Reply:
OK

Reply:
@jongwu I have adjusted it, please take a look.